### ISSUE TITLE
new tests not recognised until project reload

### ISSUE THREAD

**Issue by C0DK** [ionide/ionide-vscode-fsharp#1892]
### Describe the bug
I am running a Fsharp console project on VS Code using the Ionide plugin. I added a test file using Nunit. However, when I add a new test to the test file, the test case only gets recognized when I reload the window using the command palette.

(Copy of [this issue on stack overflow](https://stackoverflow.com/questions/76148568/fsharp-ionide-on-vs-code-new-tests-not-recognised-until-project-reload))

### Steps to reproduce
Add additional unit tests in a module utilizing NUnit
### Expected behavior
Test Explorer shows newly added test
### Screenshots
![image](https://github.com/ionide/ionide-vscode-fsharp/assets/9048078/ec90fe49-11a4-474d-ab93-d4d99bcbb922)
## Machine info
 - OS: Linux (PopOs)
 - .NET SDK version: 6.0.116
 - Ionide version: 7.7.0
 - vscode version 1.75.1 (441438abd1ac652551dbe4d408dfcec8a499b8bf)

### Additional context
N/A

**Comment by baronfel** [comment]
Does the new file get recognized as part of a project in the Solution Explorer? Test parsing only happens on files that are part of a Project, so if that step isn't happening that would explain this behavior.

**Comment by C0DK** [comment]
Yes. If i have a file, right after starting VSCode the existing tests are shown; if I add a new test, the new one is not as per the following screenshot where I just added `Example 3`
![image](https://github.com/ionide/ionide-vscode-fsharp/assets/9048078/4c7bf7a9-50d5-4837-9bd5-b80ba36f00c6)

**Comment by jkone27** [comment]
my tests are not reckognized by vscode all times too, kind of struggling on this one too

**Comment by jkone27** [comment]
any update on this one? i think sometimes the test explorer hangs, is there a timeout setting to introduce and force refresh if timeout expires or some other "forced" fix to ionide tests? even in sample solutions i have this, most of my solution are with both C# and F# projects...

**Comment by jkone27** [comment]
sorry @C0DK i see in your case you missed the unit or empty parens `()` after your function test definition, i believe sometimes F# cannot reckognized function missing 1 arg as it thinks you want to partially apply them, maybe a fix could be prevent this from happening from anything that has a [<Fact>] or other test attributes.... but might need some FsAutocomplete work i suppose or some other way to generate warnings...

```fsharp

[<Fact>]
let ``example 3 (wrong)`` = // unit
       Assert.True(true) // this is not a valid test for XUnit as it expects an "action" with signature `unit -> unit` in this case

[<Fact>]
let ``example 3 (ok, add unit)`` () = // unit --> unit
       Assert.True(true) // this is not a valid test for XUnit as it expects an "action" with signature `unit -> unit` in this case

```

i guess this can be closed?
